projects/Mallard/src/screens/home-screen.tsx

```diff
@@ -218,6 +232,9 @@ const IssueListFooter = ({ navigation }: NavigationInjectedProps) => {
             {isUsingProdDevtools ? (
                 <GridRowSplit>
                     <Button
+                        accessible={true}
+                        accessibilityLabel={'Go to latest button'}
+                        accessibilityHint={'Navigates to the latest edition'}
```

Comment:
Using consistent pattern of setting accessibility in this file would be better.

Instead of using `{'some value'}` using `"some value"`

Reply:
Good spot, thanks @mohammad-haque!